pom.xml

```diff
@@ -19,6 +19,26 @@
             </plugin>
         </plugins>
     </build>
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
```

Comment:
Move versions to a property section of POM file. This way, POM will be more manageable.